src/modules/rppt_tensor_audio_augmentations.cpp

```diff
@@ -268,3 +272,46 @@ RppStatus rppt_resample_host(RppPtr_t srcPtr,
         return RPP_ERROR_NOT_IMPLEMENTED;
     }
 }
+
+/********************************************************************************************************************/
+/*********************************************** RPP_GPU_SUPPORT = ON ***********************************************/
+/********************************************************************************************************************/
+
+#ifdef GPU_SUPPORT
+
+/******************** resample ********************/
+RppStatus rppt_resample_gpu(RppPtr_t srcPtr,
+                            RpptDescPtr srcDescPtr,
+                            RppPtr_t dstPtr,
+                            RpptDescPtr dstDescPtr,
+                            Rpp32f *inRateTensor,
+                            Rpp32f *outRateTensor,
+                            Rpp32s *srcDimsTensor,
+                            RpptResamplingWindow &window,
+                            rppHandle_t rppHandle)
+{
+#ifdef HIP_COMPILE
+    if (srcDescPtr->dataType == RpptDataType::F32)
+    {
+        hip_exec_resample_tensor(static_cast<Rpp32f*>(srcPtr),
+                                 srcDescPtr,
+                                 static_cast<Rpp32f*>(dstPtr),
+                                 dstDescPtr,
```

Comment:
What if user gives srcDescPtr->numDims or dstDescPtr->numDims as not 2 or 3? Pls add in a check and appropriate error code return on both HOST and HIP